_includes/icml/2020/about.icml.html

```diff
@@ -0,0 +1,380 @@
+<!--        <li>Shruti Jadon, Brown University(Rhode Island Hospital)</li>-->
+<!--        <li>Shuai Tang, University of California, San Diego</li>-->
+<!--        <li>Sufeng Niu, Linkedin</li>-->
+<!--        <li>Vinay PRABHU, UnifyID</li>-->
+<!--      </ul>-->
+      
+      <p>Interested in becoming a reviewer? Join our Program Committee as a
+      reviewer  
+<!--      <a class="project-link" href="/icml-2019-committee">Join Committee</a>-->
+        <a  href="">Join Committee</a>
+      </p>
+    </div>
+
+    
+    <div id="workshopVolunteer" style="color: black">
+      <hr>
+      <h3>Volunteers Appreciated</h3>
+      <p>Interested in volunteering with LatinX in AI during the event? Please
+      complete the volunteer signup form <a
+        href="">here.</a></p>
+      <hr>
+    </div>
+      
+
+    <div id="workshopLogistics">
+      <h3>Logistics</h3>
+        
+      <details>
+        <summary>Attending ICML</summary>
+        <p>Please notes that an abstract acceptance in LXAI and/or
+        registration to attend LXAI does not guarantee access to ICML. You
+        have to register to attend ICML separately (<a target='_blank'
+          href="https://icml.cc/Register/view-registration">
+          ICML registration</a>).
+        </p>
+
+        <p>As a result, we would like to inform you that ICML registration
+        will sell out quickly. If you are planning to attend ICML, we want to
+        encourage you to register now.  You can cancel your registration for
+        a full refund until June 29th if you end up not being able to
+        attend. (<a target='_blank'
+          href="https://icml.cc/Help/CancellationPolicy">
+          cancellation policy</a>).
+        </p>
+
+        <p>We also encourage you to book your accommodation as the discounted
+        room blocks are being filled up quickly.</p>
+      </details>
+
+      <details>
+        <summary>Accommodations</summary>
+        <p>You may take advantage of ICML group hotel rates, provided <a
+          target='_blank'
+          href="https://icml.cc/Conferences/2020/Hotels">here</a>. In the past,
+        workshop participants have also used <a target='_blank'
+          href="http://www.airbnb.com/">Airbnb</a> and hostels.
+        </p>
+
+        <p>Hotel cancellation policy should be checked with the hotels.</p>
+
+        <p>LXAI is not responsible for information provided on external
+        websites.</p>
+      </details>
+
+      <details>
+        <summary>More Visa Information</summary>
+        <p>A visa invitation letter comes with the ICML registration. If you
+        don’t have the ICML visa invitation letter, we can also provide you
+        Invitation letters upon successful registration to the LXAI workshop.
+        To know more about the appointment waiting times for interviews <a
+          href="https://travel.state.gov/content/travel/en/us-visas/visa-information-resources/wait-times.html">check
```

Comment:
This link is for visa requirements in the US. Should be a link to visa requirements in Austria.